src/skin/tooltips.cpp

```diff
@@ -637,4 +652,102 @@ void Tooltips::addStandardTooltips() {
     add("starrating")
             << tr("Star Rating")
             << tr("Assign ratings to individual tracks by clicking the stars.");
+
+    // Effect Unit Controls
+    add("EffectUnit_clear")
+            << tr("Clear Unit")
+            << tr("Clear effect unit.");
+
+    add("EffectUnit_enabled")
+            << tr("Toggle Unit")
+            << tr("Toggle effect unit.");
```

Comment:
[Post fader effects] (https://bugs.launchpad.net/mixxx/+bug/1370837) bug is on LP